packages/core/src/types/ad.ts

```diff
@@ -17,39 +17,40 @@ import { LbfgsParams } from "./state";
 // (var and node are used interchangeably)
 
 export interface IEdgeAD {
-  node: VarAD;
+    node: VarAD;
 
-  // Function "flowing down" from parent z (output, which is the node stored here) to child v (input), dz/dv
-  // Aka how sensitive the output is to this input -- a function encoded as a computational graph fragment
-  sensitivityNode: MaybeVal<VarAD>;
+    // Function "flowing down" from parent z (output, which is the node stored here) to child v (input), dz/dv
+    // Aka how sensitive the output is to this input -- a function encoded as a computational graph fragment
+    sensitivityNode: MaybeVal<VarAD>;
 }
 
 export type EdgeAD = IEdgeAD;
 
 export interface IVarAD {
-  val: number; // The value of this node at the time the computational graph was created. This is mostly unused, since most values are compiled out except for leaf nodes
-
-  valDone: boolean; // formerly used to cache energy values in the computational graph in evalEnergyOnGraph; TODO: can be removed if evalEnergyOnGraph is removed
-
-  tag: "custom";
-  metadata: string; // Used for storing the kind of weight
-  op: string;
-  isCompNode: boolean; // comp node (normal computational graph) or grad node (node in computational graph for gradient)
-  isInput: boolean; // These inputs need to be distinguished as bindings in the function (e.g. \x y -> x + y)
-  parents: EdgeAD[]; // The resulting values from an expression. e.g. in `z := x + y`, `z` is a parent of `x` and of `y`
-  children: EdgeAD[];
-  parentsGrad: EdgeAD[]; // The resulting values from an expression. e.g. in `z := x + y`, `z` is a parent of `x` and of `y`
-  childrenGrad: EdgeAD[];
-  gradVal: MaybeVal<number>;
-  gradNode: MaybeVal<VarAD>;
-  index: number; // -1 if not a leaf node, 0-n for leaf nodes (order in the leaf node list) so we know how to pass in the floats
-
-  debug: boolean; // If true, this prints node debug info on evaluation
-  debugInfo: string;
-
-  nodeVisited: boolean;
-  // Now used to track whether this node (and its children) has already been computed in the codegen
-  name: string; // Name of cached value for this node in codegen (e.g. `const x3 = x1 + x2;` <-- name of node is `x3`)
+    val: number; // The value of this node at the time the computational graph was created. This is mostly unused, since most values are compiled out except for leaf nodes
```

Comment:
Prettier should fix this. Our tab setting is __2__ spaces instead of 4. Can you double-check if prettier runs? You should see some messages from it when you run the pre-commit hook.

Reply:
For some reason the pre-commit hook is not running again. I've tried doing `git config --unset core.hookspath` in a couple different dirs and re-committing this file, but with no luck. Any suggestions?

Reply:
Still no luck with this. After reading some SO posts, I updated my git version to to 2.32 (on mac). Here is the result of running `git config core.hooksPath` in ROOT:

`ROOT/.git/hooks/` (where ROOT is an absolute path) 

I ran `yarn-test` by hand, which passed, and also ran `prettier` by hand (on `ad.ts`, the file with wrong spacing).

Reply:
Have you tried doing `yarn clean` and `yarn`?

Reply:
Just tried it, no luck.